api/python/provisioner/commands/sw_rollback.py

```diff
@@ -0,0 +1,143 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+'''Defines sw_rollback module to perform CORTX software rollback'''
+
+import logging
+from typing import Type
+
+from ._basic import RunArgs, CommandParserFillerMixin
+from ..vendor import attr
+from ..config import ALL_MINIONS
+from ..errors import (
+    BadPillarDataError,
+    SWRollbackError,
+)
+from ..pillar import (
+    PillarKey,
+    PillarResolver
+)
+from ..salt import (
+    StatesApplier,
+    YumRollbackManager,
+    local_minion_id
+)
+from ..commands import _apply_provisioner_config
+from ..salt_master import config_salt_master
+from ..salt_minion import config_salt_minions
+from .. import inputs, values
+
+logger = logging.getLogger(__name__)
+
+
+@attr.s(auto_attribs=True)
+class RunArgsSWRollback:
+
+    '''Specify usage of attributes required for rollback'''
+
+    target_version: str = attr.ib(
+        metadata={
+            inputs.METADATA_ARGPARSER: {
+                'help': "CORTX version to rollback "
+                "in case yum rollback is not performed"
+            }
+        },
+        default=None
+    )
+    targets: str = RunArgs.targets
+
+
+@attr.s(auto_attribs=True)
+class SWRollback(CommandParserFillerMixin):
+
```

Comment:
you don't need empty line here, please refer to mentioned PEP 257 and [numpy](https://numpydoc.readthedocs.io/en/latest/index.html) which we decided to follow

Reply:
Addressed